channels-rke2.yaml

```
@@ -352,6 +352,24 @@ releases:
       rancher-vsphere-csi:
         repo: rancher-charts
         version: 100.1.0+up2.3.0
+  - version: v1.21.11+rke2r1
+    minChannelServerVersion: v2.6.4-alpha1
```

```suggestion
    minChannelServerVersion: v2.6.3-alpha1
```

This is based off a change that was made to dev-v2.6 for the rancher release. Rebasing your branch should pull this in for v1.21.10+rke2r2 so this change is just to make it consistent with that.